bloom.go

```diff
@@ -8,6 +8,7 @@ import (
 type BitSetProvider interface {
 	Set([]uint) error
 	Test([]uint) (bool, error)
+	Delete() error
```

Comment:
I don't think Delete should be a method of this interface. Like New, it's an implementation-specific behavior that does not apply to all implementations. RedisBitSet can keep the method, but other bitsets don't need it.